Makefile

```diff
@@ -465,14 +465,14 @@ save:
 	cp Makefile ./user_projects/$(PROJ)
 	cp VERSION.txt ./user_projects/$(PROJ)
 	cp ./config/* ./user_projects/$(PROJ)/config
```

Comment:
Might as well do the same for the config folder, too, yeah?

Reply:
I don't think the config folder needs a deeper folder structure. 
There are usually just files in the config folder (PhysiCell_settings.xml, cell_rules.csv, cells.cvs). I don't need changes there.

Reply:
Sure. But since we're here, let's also change the behavior for the config folder, yeah?

Reply:
on one hand, I don't care, on the other and I don't see a reason why to change that.
do you have an example, in which case you would need a deeper folder structure in config? 
do you have a user project that has configurations that have to be stored in a folder structure?
the case with the custom_modules is different. i have two modules I wrote, and each of these modules is in a folder, and it would be very tedious and result in a chaotic codebase, if i had to pack these modules into files only, no folders allowed.

Reply:
There is a need: we always put our boolean models in a folder.

Reply:
@MathCancer befor you approve, I think I will then add @vincent-noel request too?!

Reply:
Thanks Elmar !